<ui version="4.0" >
 <class>SvnSSLTrustDialog</class>
 <widget class="QWidget" name="SvnSSLTrustDialog" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>461</width>
    <height>407</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>SSL Server Certificate</string>
  </property>
  <layout class="QGridLayout" >
   <property name="leftMargin" >
    <number>9</number>
   </property>
   <property name="topMargin" >
    <number>9</number>
   </property>
   <property name="rightMargin" >
    <number>9</number>
   </property>
   <property name="bottomMargin" >
    <number>9</number>
   </property>
   <property name="horizontalSpacing" >
    <number>6</number>
   </property>
   <property name="verticalSpacing" >
    <number>6</number>
   </property>
   <item row="2" column="0" colspan="2" >
    <widget class="QTreeWidget" name="treeWidget" >
     <property name="indentation" >
      <number>0</number>
     </property>
     <property name="columnCount" >
      <number>2</number>
     </property>
     <column>
      <property name="text" >
       <string>Item</string>
      </property>
     </column>
     <column>
      <property name="text" >
       <string>Value</string>
      </property>
     </column>
    </widget>
   </item>
   <item row="3" column="0" >
    <widget class="QCheckBox" name="permanentCheck" >
     <property name="text" >
      <string>Save Permanently</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2" >
    <widget class="QLabel" name="msgLabel" >
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
   <item row="1" column="0" colspan="2" >
    <widget class="QLabel" name="reasonLabel" >
     <property name="sizePolicy" >
      <sizepolicy vsizetype="Preferred" hsizetype="Preferred" >
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text" >
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
